extension/lib/DoorhangerStorage.jsm

```diff
@@ -32,10 +32,10 @@ this.DoorhangerStorage = {
       timestamp: Date.now(),
       neverAgain: !!neverAgain,
     };
-    return await this.getStore().put(data, hostname);
+    return this.getStore().put(data, hostname);
   },
 
   async unsetStats(hostname) {
-    return await this.getStore().delete(hostname);
+    return this.getStore().delete(hostname);
```

Comment:
ESLint was saying these two lines (L35 and L39) had redundant `await` statements:

> _"Redundant use of `await` on a return value"_  [no-return-await](https://eslint.org/docs/rules/no-return-await)